content/docs/user-guide/external-dependencies.md

```diff
@@ -54,11 +54,13 @@ $ dvc run -n download_file
 
 ```dvc
 $ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
+          -d ssh://user@example.com/home/shared/data.txt \
           -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
+          scp user@example.com/home/shared/data.txt data.txt
 ```
 
+> Please notice `/home/...` is an absolute path from the remote system's root.
```

Comment:
Strictly speaking this is not about system's root, but rather sftp root directory, which doesn't always match system's root.

Reply:
Note updated.